workflow/cron/operator.go

```diff
@@ -321,3 +342,16 @@ func (woc *cronWfOperationCtx) reportCronWorkflowError(conditionType v1alpha1.Co
 	})
 	woc.metrics.CronWorkflowSubmissionError()
 }
+
+func inferScheduledTime() time.Time {
+	// Infer scheduled runtime by getting current time and zeroing out current seconds and nanoseconds
+	// This works because the finest possible scheduled runtime is a minute. It is unlikely to ever be used, since this
+	// function is quickly supplanted by a deterministic function from the cron engine.
+	log.Infof("inferred scheduled time")
```

Comment:
good idea to log this I think we should log the inferred time too

Reply:
Will add